### ISSUE TITLE
[lint] unsafe_html

### ISSUE THREAD

**Issue by evanweible-wf** [Workiva/workiva_analysis_options#143]
http://dart-lang.github.io/linter/lints/unsafe_html.html

- Infractions should be rare
- Having to explicitly ignore valid usages is a good practice, given the security concern
- (As of Nov 15) Number of infractions at Workiva: **16**

**Comment by dustyholmes-wf** [comment]
@evanweible-wf it looks like since your evaluation google has added several more reasons why this would be triggered to the list. They do not seem to provide any guidance on alternate solutions though. In trying to get 2.13 ready, I have found 6 new instances in DPC and ignored them since I'm not sure what the correct resolution is. Makes me wonder about the value of the lint.

**Comment by evanweible-wf** [comment]
Yeah, that's a good point. Sounds like this would be better suited as a recommended lint and people can ignore individual instances where it's been evaluated and OK'd.
